### PR TITLE
Checkstyle: Fix member name violations

### PR DESCRIPTION
**Pull request by ssoloff**
This PR fixes violations of the Checkstyle MemberName rule in UI and non-serializable types under the `g.s.engine.framework.startup` package.

### CODE REVIEW

File: src/main/java/games/strategy/engine/framework/startup/ui/ClientOptions.java [hunk 1]
```diff
@@ -103,18 +102,15 @@ private void layoutComponents() {
...
 
   public boolean getOkPressed() {
     return okPressed;
   }
-
-  private final Action m_okAction = SwingAction.of("Connect", e -> {
-    setVisible(false);
-    okPressed = true;
-  });
-  private final Action m_cancelAction = SwingAction.of("Cancel", e -> setVisible(false));
```
Comment (ssoloff) [inline src/main/java/games/strategy/engine/framework/startup/ui/ClientOptions.java]: These fields were only used in one place so I just inlined them above.

File: src/main/java/games/strategy/engine/framework/startup/ui/ServerOptions.java [hunk 2]
```diff
@@ -157,13 +159,6 @@ private void setWidgetActivation() {
     passwordField.setBackground(backGround);
   }
 
-  private final Action m_okAction = SwingAction.of("OK", e -> {
-    setVisible(false);
-    okPressed = true;
-  });
-
-  private final Action m_cancelAction = SwingAction.of("Cancel", e -> setVisible(false));
-
```
Comment (ssoloff) [inline src/main/java/games/strategy/engine/framework/startup/ui/ServerOptions.java]: These fields were only used in one place so I just inlined them above.

**Comment by ssoloff** [comment]
Hmm, I was purposefully avoiding making the same renames as in #2509.  I guess I missed some. 😞

**Comment by RoiEXLab** [comment]
Didn't check it explicitly, just assuming this, because the file was heavily modified 🤷‍♂️

**Pull request merged**